backend/GymBeam/Controllers/AuthenticationController.cs

```diff
@@ -24,14 +24,15 @@ public AuthenticationController(IConfiguration configuration)
 
         [HttpPost("Register")]
         [AllowAnonymous]
-        public ActionResult<UserResponse> Register([FromBody] UserRequest dto)
+        public ActionResult<UserResponse> Register([FromBody] RegisterRequest dto)
         {
             return new UserResponse
             {
                 Id = 34,
-                Name = dto.Name,
+                Name = dto.Username,
                 DisplayName = dto.DisplayName,
-                Role = dto.Role
+                Role = "user",
```

Comment:
Please use constant instead of hardcoded string

Reply:
For now it can be a temporary "user". It will be mapped in the handler from Domain.Roles to UserResponse.Role